src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -9346,50 +9403,63 @@ public static void M2(object o, bool b1, bool b2, bool b3)
             var compVerifier = CompileAndVerify(compilation, expectedOutput: "abc");
             compVerifier.VerifyIL("Program.M2",
 @"{
-  // Code size       79 (0x4f)
-  .maxstack  1
-  .locals init (Generic<object, System.ValueTuple<int, int>> V_0, //g
-                Generic<dynamic, System.ValueTuple<int, int>> V_1, //g
-                object V_2,
-                object V_3)
+  // Code size       98 (0x62)
```

Comment:
>Code size       98 (0x62) [](http://example.com/codeflow?start=5&length=25)

Why did the IL change for this case? `M2()` doesn't seem to include any shared `when` clauses.

Reply:
`b2` (in second `case`) is a shared when clause.